runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -1140,10 +1195,24 @@ int chpl_comm_numPollingTasks(void) {
   return 1;
 }
 
-void chpl_comm_make_progress(void) { }
+
+inline
```

Comment:
Does marking this inline do anything?

Reply:
Yes, it'll get inlined into the local calls.  I decided this was probably worthwhile, to reduce the latency of the check for auto vs. manual progress.

Reply:
I might just make this a static inline function and remove `chpl_comm_make_progress()` -- https://github.com/chapel-lang/chapel/issues/6349